spec/features/details_on_index_by_admins_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.feature "DetailsOnIndexByAdmins", type: :feature do
+  let!(:user) { create :user, email: "example22@o2.pl", password: "password" }
+
+  background do
+    sign_in_admin
+    visit root_path
+  end
+
+  scenario do
+    expect(page).to have_xpath("//a[@title='delete']")
```

Comment:
It's better to use css classes then xpaths in feature specs. You can test if thrash icon is visible.